src/slic3r/GUI/ObjectDataViewModel.cpp

```diff
@@ -1048,20 +1048,20 @@ wxDataViewItem ObjectDataViewModel::Delete(const wxDataViewItem &item)
         ObjectDataViewModelNode* last_instance_node = node_parent->GetNthChild(0);
         PrintIndicator last_instance_printable = last_instance_node->IsPrintable();
         node_parent->GetChildren().Remove(last_instance_node);
-        delete last_instance_node;
         ItemDeleted(parent, wxDataViewItem(last_instance_node));
+        delete last_instance_node;
```

Comment:
What wxwidgets does here
https://github.com/wxWidgets/wxWidgets/blob/dac1996ebe162386071c0b9148079fabf3a950e3/src/common/datavcmn.cpp#L3139-L3147
is delete the pointer first then call  `ItemDeleted`, I guess it's not a bug here too.